.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "makefile.extensionOutputFolder": "./.vscode"
```

Comment:
Are you commiting .vscode on purpose or should that go to .gitignore?

Reply:
Committing it on purpose. I was going to include it in `.gitignore`, but after seeing this in a number of repos I’ve worked in, I decided to leave it, as it may be useful for others. Open to your thoughts on this, though.

Reply:
Alright, I'm ok with that.